From 849c907d5e8ebcf43b100783813266a6160d9aa9 Mon Sep 17 00:00:00 2001
From: Danny Park <danny.park@outlook.com>
Date: Fri, 22 Feb 2019 15:29:05 -0500
Subject: [PATCH 4/4] Add CONFIG_SYS_TEXT_BASE a NEW required option in 2017.3.
  Also add CONFIG_SYS_MALLOC_F_LEN in case it is required.

---
 configs/zynq_red_pitaya_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/configs/zynq_red_pitaya_defconfig b/configs/zynq_red_pitaya_defconfig
index 88a7eeb..7a46c92 100644
--- a/configs/zynq_red_pitaya_defconfig
+++ b/configs/zynq_red_pitaya_defconfig
@@ -1,6 +1,8 @@
 CONFIG_ARM=y
 CONFIG_SYS_CONFIG_NAME="zynq_red_pitaya"
 CONFIG_ARCH_ZYNQ=y
+CONFIG_SYS_MALLOC_F_LEN=0x800
+CONFIG_SYS_TEXT_BASE=0x4000000
 CONFIG_DEFAULT_DEVICE_TREE="zynq-red-pitaya"
 # CONFIG_FIT=y
 # CONFIG_FIT_VERBOSE=y
-- 
2.7.4

